docs/getting-started.md

```diff
@@ -29,6 +29,26 @@ sudo podman run --pull=always --privileged --rm \
     install /dev/vdb -i config.ign
 ```
 
+You may also add the following alias your shell configuration:
+
+```sh
+alias coreos-installer='podman run --pull=always            \
+                        --rm --interactive                  \
+                        --security-opt label=disable        \
+                        --volume ${PWD}:/pwd --workdir /pwd \
+                        quay.io/coreos/coreos-installer:release'
+```
+
+If you want to be able to cancel `coreos-installer` downloads, you
+will have to add the `--tty` option to your podman comand line or shell alias:
```

Comment:
I'd advise against using a Bash alias for `coreos-installer` due to this reason. A script however can easily solve this by simply checking whether a `tty` is connected to the script and choose accordingly:

```sh
#!/bin/sh
if tty -s; then
    podman run … --tty …
else
    podman run … # without --ty
fi
```

Reply:
By the way: This more or less also addresses @lucab's concerns (at least if I understand him right), because with a script it's a matter of `$PATH` which `coreos-installer` is used (i.e. the "usual" way), instead of an alias overruling `$PATH`.